app/gen-server/entity/Activation.ts

```diff
@@ -22,6 +22,9 @@ export class Activation extends BaseEntity {
   @Column({name: 'updated_at', default: () => "CURRENT_TIMESTAMP"})
   public updatedAt: Date;
 
+  @Column({name: 'enabled_at', type: nativeValues.dateTimeType, nullable: true})
+  public enabledAt: Date|null;
```

Comment:
Have you considered, changing this name to `trialStart`, or `firstEnabledAt` ? Otherwise can you add comments what it does and how it is meant to be used (for example what should happen after toggling twice)?

Reply:
I've been [externally using](https://github.com/gristlabs/grist-core/pull/1095#issuecomment-2257475350) the phrase "enable" to separate the process of enabling enterprise from activating enterprise, so I opted to add a comment instead.

Reply:
Looks good. Can you extend it to include description how it works when user toggles enterprise features multiple times? But it can be extended in the following PR, if you prefer.

Reply:
Sure, sorry, added the word "first" to explain.